queries.md

```diff
@@ -6,27 +6,70 @@
 
  - **`query`**: {name: 'Facebook'}
  
+
  ## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
 
  - **`query`**: {category_code: 'web'}
  - **`projection`**: {name: 1, _id: 0}
 
+
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
 
+- **`query`**:{name: 'Twitter'}
+- **`projection`**: {$and: [{category_code: 1}, {founded_year: 1}]}
+
+
 ## 4. Find all the companies who have `web` as their **category_code**, but limit the search to 50 companies.
 
+- **`query`**: {category_code: 1}
+- **`limit`**: 50 
+
+
 ## 5. Find all the companies which **category_code** is 'enterprise' and have been founded in 2005. Retrieve only the `name`, `category_code` and `founded_year` fields.
 
+- **`query`**: {$and: [{category_code: 'enterprise'}, {founded_year: '2005}]}
+- **`projection`**: {$and: [{name: 1}, {category_code: 1}, {found_year: '2005'}]}
```

Comment:
In the projection you would only use 0 or 1. 1 if you want to show the field, 0 if you don't